Developer Samples/WindowsContainers/readme.md

```diff
@@ -0,0 +1,163 @@
+# InRule Docker images
+
+## Image descriptions
```

Comment:
If you clicked them in a browser, they'll 404 until the PR is accepted at least :)
Maybe they should point to the README instead of just the directory?

Reply:
these also dont work in your branch, maybe im missing something on these.

Reply:
when I CTRL+Click in VSCode, it opens up the README in the linked directory. I'll change the links to explicitly point to the README docs instead of the naked directory

Reply:
turns out I was inadvertently specifying absolute paths. Missed the leading '/'!